include/ocpp/common/websocket/websocket_libwebsockets.hpp

```diff
@@ -100,6 +106,12 @@ class WebsocketTlsTPM final : public WebsocketBase {
     std::queue<std::string> recv_message_queue;
     std::condition_variable recv_message_cv;
     std::string recv_buffered_message;
+
+    std::unique_ptr<std::thread> deferred_callback_thread;
+    std::queue<std::function<void()>> deferred_callback_queue;
+    std::mutex deferred_callback_mutex;
+    std::condition_variable deferred_callback_cv;
+    bool stop_deferred_handler;
```

Comment:
For the sake of correction, since this variable is used from multiple threads, let's keep the mode we used bools in the .cpp file and use a 'std::atomic_bool' here.